code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -210,3 +210,11 @@
 	materials = list(/datum/material/biomass = 100)
 	build_path = /obj/item/reagent_containers/food/condiment/soymilk
 	category = list("initial","Food")
+
+/datum/design/tape
+	name = "Roll of Tape"
+	id = "tape"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass = 50)
+	build_path = /obj/item/stack/tape
+	category = list("initial","Organic Materials")
```

Comment:
Higher tier rather than baseline?

Reply:
I feel like requiring research would kind of defeat the purpose of makeshift shoddy items.